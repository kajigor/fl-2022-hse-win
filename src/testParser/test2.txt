start (ABVC);
(AC) -> (AD)(BD);
(AC) -> eps;
(AC) -> [BD] | [CD];