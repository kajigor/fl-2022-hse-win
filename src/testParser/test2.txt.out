Start nonterminal: ABVC
(AC) -> (AD)(BD)
(ABVC) -> eps
(AC) -> [BD] | [CD]
contextually free grammar in Chomsky's normal form 