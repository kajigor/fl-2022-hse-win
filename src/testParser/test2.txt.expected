Start nonterminal: ABVC
(AC) -> (AD)(BD)
(AC) -> eps
(AC) -> [BD] | [CD]
contextually free grammar not in Chomsky normal form